code/modules/bomberman/bomberman.dm

```diff
@@ -1165,7 +1165,7 @@ var/global/list/arenas = list()
 		for(var/atom/movable/AM in T)
 			AM.areaMaster = get_area_master(T)
 		if(open_space && (under.name == "Space"))
```

Comment:
After checking, under.name seems to be kind of a blueprint for building the arena, I'll just look to see if I should change it to say Base Turf